pkg/iter/iterator_test.go

```diff
@@ -154,9 +154,10 @@ func TestIteratorMultipleLabels(t *testing.T) {
 
 type generator func(i int64) logproto.Entry
 
+// nolint
 func mkStreamIterator(numEntries int64, f generator, labels string) EntryIterator {
 	entries := []logproto.Entry{}
-	for i := int64(0); i < numEntries; i++ {
+	for i := int64(0); i < 10; i++ {
```

Comment:
This one is bugging me a little, I think I would //nolint it or use the `const testSize` in here (and maybe remove numEntries from the signature) instead of hardcoding it to 10.
I think I would vote to just //nolint it

Reply:
yeah I'll review that one.